tests/test_main.py

```diff
@@ -94,41 +107,15 @@ def test_text() -> None:
            "proclaimed on December 30th, 1922."
     assert app.main.text(
         test
-    ) == {"text": test, "mp3": app.main.tts_to_mp3(test)}
-
-
-# def test_text() -> None:
-#     """Test that plain text can be converted into a .mp3 file."""
-#     test = "The Union of Soviet Socialist Republics was" \
-#            "proclaimed on December 30th, 1922."
-#     assert app.main.text(
-#         test
-#     ) == {"text": test, "mp3": app.main.tts_to_mp3(test)}
-
-# def test_convert_text() -> None:

```

Comment:
Don't delete this test. It's an important integration test being included in the final report.

Reply:
If this what you are pointing out, test_convert_text() is not in the Text Plan from what I see and test_text() is already built into test_main.py. There is no need to keep this commented-out version.